tests/base/test_port_forwarding.py

```diff
@@ -52,6 +52,14 @@ def _create_nginx_container(number: int) -> DerivedContainer:
         forwarded_ports=[
             PortForwarding(container_port=80),
             PortForwarding(container_port=443),
+        ]
+        # create an insane number of additional port forwards, hopefully
+        # catching any race conditions or something similar that could cause
+        # pytest_container to give two containers the same port
+        + [PortForwarding(container_port=n) for n in range(500, 1000)]
```

Comment:
While this should work, couldn't we create a hash value from the container name and use this (modulo a given range) to avoid port collisions? That should avoid race conditions between different containers, given that the hash function is sufficiently randomized.

Reply:
Huh? This is a test, why should we randomize these ports at all?